addons/l10n_id_efaktur/models/res_partner.py

```diff
@@ -8,7 +8,6 @@ class ResPartner(models.Model):
     """Inherit res.partner object to add NPWP field and Kode Transaksi"""
     _inherit = "res.partner"
 
-    country_code = fields.Char(related='country_id.code', string='Country Code')
```

Comment:
this kind of cleanup has nothing to do with your task.

Reply:
It does actually, because I add the field in base/res_partner.py, so if I leave this there, a warning is raised.

Reply:
Also if the field is still called country_code?

Reply:
Yes it's still called country_code. I think it's better to have it at a higher level in the hierarchy since we use it more and more, especially in the edis.